client/src/components/PageLogin/PageLogin.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { useState } from 'react'
+import { doLogInSaga } from '../../redux/actions/actionLogIn'
+import { useDispatch } from 'react-redux';
+
+const PageLogin = () => {
+
+  const [form, setForm] = useState({
+    email: '',
+    password: ''
+  })
+
+  const dispatch = useDispatch()
+
+  const onChangeForm = (e) => {
```

Comment:
These handlers get replicated. It looks like we need to move them to a separated hook and use it in different components. https://trello.com/c/RjnU29Eq/20-useform-hook

Reply:
fixed later. added to TODO. Now I don't understand how can I do it. I'll need to read about it.